src/CONST.ts

```diff
@@ -83,6 +83,14 @@ const CONST = {
 
     MERCHANT_NAME_MAX_LENGTH: 255,
 
+    CARD_STATE: {
+        OPEN: 3,
+        NOT_ACTIVATED: 4,
+        DEACTIVATED: 5,
+        CLOSED: 6,
+        SUSPENDED: 7,
+    },
```

Comment:
Could you please use `CONST.EXPENSIFY_CARD.STATE` instead

Reply:
yes ofc. It wasn't in main at the time of creating this